packages/nitro-protocol/gas-benchmarks/gas.ts

```diff
@@ -107,9 +107,9 @@ export const gasRequiredTo: GasRequiredTo = {
       challengeJ: 101068,
       challengeX: 92757,
       transferAllAssetsL: 65462,
-      claimG: 82369,
-      transferAllAssetsX: 114930,
-      total: 642227,
+      claimG: 94758,
+      transferAllAssetsX: 44478,
```

Comment:
Does this large change in gas reflect the fact that assets are being transferred to different addresses?.

Reply:
The large change here reflects the fact that X is actually not funded at all so cannot afford any payouts. 

At this point the benchmarking script is still wrong (in the sense that the channels have been setup incorrectly), whereas the on chain code has been (partly) fixed.